src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
     }
```

Comment:
Avoid using static methods as per the checklist. Instead, you should create an instance of `ColorSupplier` in the class that needs it. Moreover, using `new Random()` inside the method call will create a new `Random` object each time the method is called, which is unnecessary. It's better to have a single `Random` instance as a private field of the class. Lastly, use `name()` instead of `toString()` to get the name of the enum constant.